lib/oauth2/client.rb

```diff
@@ -4,7 +4,7 @@
 module OAuth2
   # The OAuth2::Client class
   class Client # rubocop:disable Metrics/ClassLength
-    RESERVED_PARAM_KEYS = ['headers', 'parse'].freeze
+    RESERVED_PARAM_KEYS = %w[headers parse].freeze
```

Comment:
make Rubcop happy. Could make this a separate change if folks would prefer.

Reply:
Rubocop updates are always great.  Clean things up bit by bit as we go.